Documentation/build-parameters.rst

```diff
@@ -0,0 +1,98 @@
+Build Parameter Reference
+=========================
+
+**For Developers:** if you add a CMake parameter, make sure to document it here.
+
+For a good overview and easy, we recommend the use of ``ccmake``. To do so, simply replace the use of ``cmake`` by ``ccmake``.
+(i.e. add a "c" in front of the command)
+Alternatively, you may also modify the ``CMakeCache.txt`` in your build directory directly (for it to exist, ``cmake`` needs to be run first).
+
+Equation-specific parameters
```

Comment:
Can we move it below CMAKE_BUILD_TYPE? As the build type is not really an equation-specific parameter.